src/commands/run.rs

```diff
@@ -168,6 +170,23 @@ pub struct RunCommand {
     )]
     wasm_timeout: Option<Duration>,
 
+    /// Enable in-process sampling profiling of the guest WebAssembly program,
+    /// and write the captured profile to the given path. The resulting file
+    /// can be viewed using https://profiler.firefox.com/.
```

Comment:
I've only been using the "local file" interface, so I didn't realize there was a way to upload a profile and share a link. I guess the way to do that is documented at https://github.com/firefox-devtools/profiler/blob/main/docs-developer/loading-in-profiles.md.

I don't see any HTTP client usage anywhere in Wasmtime except in wasi-http right now. So that seems like a relatively large change that I'd prefer to think about separately from the basic profiling implementation. It would definitely be nice for usability, though.

Reply:
Ah ok yeah if it's not easy to reuse whatever is in wasi-http I'm happy to leave this as-is